test/e2e/jobs/jobs.go

```diff
@@ -62,20 +62,20 @@ func DeleteTestInfra(subID, clusterName, location string) *types.Job {
 	return job
 }
 
-func InstallAndTestRetinaBasicMetrics(kubeConfigFilePath, chartPath string) *types.Job {
+func InstallAndTestRetinaBasicMetrics(kubeConfigFilePath, chartPath string, testNamespace string) *types.Job {
 	job := types.NewJob("Install and test Retina with basic metrics")
 
 	job.AddStep(&kubernetes.InstallHelmChart{
-		Namespace:          "kube-system",
+		Namespace:          testNamespace,
```

Comment:
Since we are allowing passing custom namespaces, why not install the retina components in a namespace and test-pods (agnhost for example) in another? Seems that would be better test.

Reply:
Yeah that makes sense! Although, after reading @matmerr comment below, Retina seems to be tightly coupled to the `kube-system` namespace. I think for the scope of this PR, I'll update the K8s manifest that the reporter was experiencing and leave the E2E tests as they were. Thanks @anubhabMajumdar!